tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -172,12 +172,13 @@ def run(self):
                                 ]
                                 if stored_peer_status != "" and \
                                     current_status != stored_peer_status:
-                                    msg = ("Status of peer: %s "
+                                    msg = ("Status of peer: %s in cluster %s "
```

Comment:
Is the UI or someone else going to extract the integration_id from msg?

Reply:
@r0h4n no, we are not going to extract anything from msg. It just to understand the user easily like this alert belongs to which cluster.

Reply:
if volume alert or brick alert came it is difficult to find the alert belongs to which cluster volume and brick, because message contains only volume or brick name.there is no cluster name.